test/functional/ui/shortmess_spec.lua

```diff
@@ -0,0 +1,40 @@
+local helpers = require('test.functional.helpers')
```

Comment:
This test isn't testing UI features; it only happens to use `Screen` to test behavior of an option. I moved it to `test/functional/options`.